wiki/Difficulties/en.md

```diff
@@ -10,18 +10,20 @@ The difficulty of a beatmap *describes* the level of skill needed to complete it
 
 ![Star rating ranges](SR_range.png "Star rating ranges")
 
-From the [Ranking Criteria](/wiki/Ranking_Criteria), a star rating of:
+For the [Ranking Criteria](/wiki/Ranking_Criteria), star ratings typically correspond to these difficulties:
 
-- 1.50 stars and below is Easy ![Easy icon](/wiki/shared/diff/easy-s.png "Easy icon")
-- 1.51 - 2.25 stars is Normal ![Normal icon](/wiki/shared/diff/normal-s.png "Normal icon")
-- 2.26 - 3.75 stars is Hard ![Hard icon](/wiki/shared/diff/hard-s.png "Hard icon")
-- 3.76 - 5.25 stars is Insane ![Insane icon](/wiki/shared/diff/insane-s.png "Insane icon")
-- 5.26 - 6.75 stars is Expert ![Expert icon](/wiki/shared/diff/expert-s.png "Expert icon")
-- 6.76 stars and above is Expert+ ![Expert+ icon](/wiki/shared/diff/expertplus-s.png "Expert+ icon")
+- ![Easy icon](/wiki/shared/diff/easy-s.png "Easy icon") Easy: 1.0★–2.0★
```

Comment:
```suggestion
- ![Easy icon](/wiki/shared/diff/easy-s.png "Easy icon") Easy: 0.0★–1.99★
```

Reply:
For the 1.0 to 0.0 change, I'd argue that *very* few maps go below 1.0★ and all typically are named "Beginner" or have special names. The graphs in ppy/osu-web#4181 show fuzziness around the borders.

As for the .99 change, I think the precision is unnecessary and arbitrary, in my opinion—it would be most correct to say <2.0★ etc., but I think just specifying ranges without providing a [strict inequality](https://en.wikipedia.org/wiki/Inequality_(mathematics)) makes reading it a tiny bit more difficult for no additional benefit.

Reply:
the .99 thing can go either way, but you should use 0 for the lower bound here because we're only concerned about the icon in this section, and 0-1 stars does use the Easy icon

Reply:
> For the 1.0 to 0.0 change, I'd argue that very few maps [...]

I suppose so, but weren't these going to list out the icons that would be used on the site? Sure, they'd be called "Beginner", but the site would still use a green icon.

> As for the .99 change, I think [...]

Probably so, but if going off of this article, would a beatmap with 2.0★ be both Easy and Normal? Again, from above, weren't these going to list out the icons that would be used on the site?

Reply:
yeah that's a good point I'd go with the .99 then. (and all of MAP's related suggestions)

also stays consistent with how the article was previously written, which seemed to serve its purpose fine

Reply:
Fair enough.